File: JavaScript/LinearSearch/linearSearch.js
const linearSearch = (arr, num) => {
	let index = -1;

	arr.forEach((n, i) => {
		if (n == num) {
			index = i;
		}
	});

	return index;
};

(function () {
	const arr = [1, 15, 547, 1864, 531, 510, 5, 64];

	console.log(linearSearch(arr, 531));
})();

File: Python/linear_search.py
#linear search
def linear_search(array,key):
           for i in range(len(array)):
                      if array[i]==key:
                                 return(i)
           else:
                      return(-1)

arr=[8,4,7,6,2,5]
key=5
i=linear_search(arr,key)
if i!=-1:
           print(key ,"is at index ",i)                          
else:
           print(key,"not found")
